Framework/Utils/src/raw-parser.cxx

```diff
@@ -51,18 +51,15 @@ WorkflowSpec defineDataProcessing(ConfigContext const& config)
 			  << dh->subSpecification << " payload size " << dh->payloadSize;
 	      }
 
-	      // there is a bug in InpuRecord::get for vectors of simple types, not catched in
-	      // DataAllocator unit test
-	      //auto data = inputs.get<std::vector<char>>(input.spec->binding.c_str());
-	      //LOG(INFO) << "data size " << data.size();
+	      auto raw = inputs.get<std::span<char>>(input.spec->binding.c_str());
```

Comment:
@sawenzel the travis check for the formatting is failing after I have removed the comments and changed this line. The suggested formatting is strange, it does not align with the code above and below anymore which went successfully through the formatting before.
Do you have an idea what can trigger clang-format to produce this formatting? Or has there be a recent change in the configuration?